test/transpile/module-resolution/project-with-dependency-chain/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "strict": true,
+        "moduleResolution": "Node",
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "target": "esnext",
+        "lib": ["esnext"],
+        "types": [],
+        "rootDir": "."
+    }
+}
```

Comment:
Maybe try another test project that uses an `module` option other than `commonJS` (e.g. `esnext)`.
Honestly not sure if that will even works with our current import/export implementation, but it would be nice to knowing any case, so we can document that limitation.